CHANGES.md

```diff
@@ -16,6 +16,10 @@ Rose release 61. This is a __beta pre-release__.
 
 For use with Cylc see [Cylc-Rose](https://github.com/cylc/cylc-rose).
 
+[#2493](https://github.com/metomi/rose/pull/2493) -
+Disable the `rosie disco` web service pending a fix to the tornado version
+conflict with cylc-uiserver, see https://github.com/metomi/rose/pull/2493
+
```

Comment:
Shouldn't it be under 2.0b3?

Reply:
Ha, I guess we didn't make any changes for b2!

Reply:
Created a b3 section for this entry.